### ISSUE TITLE
Settings not saved for user

### ISSUE THREAD

**Issue by jancborchardt** [erming/shout#542]
The settings only seem to be saved for the browser. This is a bit confusing when you switch devices or browsers as suddenly everything is back to the default again.

I’d say they should be saved for the user, as that’s the context you modify them in.

**Comment by floogulinc** [comment]
This is the intended functionality as users may want different settings on say, their phone than on their desktop, and all of the client settings are easy and quick to change. They are stored a cookie.

**Comment by YaManicKill** [comment]
So, there were some discussions in #580 about syncing settings across browsers. I think this would need to be done along with a rethink of the settings page. Here is how I think we can do it nicely:

1. Default to sync all the settings. This is what most people would expect. It could probably save them in the user settings, and if we add a section for the user settings, then that would fix #63 as well (and would be really easy to add)
2. We have an override that allows the user to override any individual setting for the browser

These could probably be done in multiple tabs ("Overall" and "For this computer") or something like that. Then it's just simple inheritance.

I think this is the best way forward for this.

**Comment by W7PEA** [comment]
Why not just move Settings to the User.JSON file? That seems much simpler.

**Comment by YaManicKill** [comment]
That does mean that you wouldn't be able to have different settings on different clients.

**Comment by W7PEA** [comment]
I wonder if there is a _real_ desire for that, I get that its possible and _might_ be something someone would want, but do people really have different settings? Maybe a Desktop\Mobile section would need to be added, but I doubt anyone wants every client to act differently by default.

I'm not saying I know for sure either, but doesn't sound like a must-have.

**Comment by jancborchardt** [comment]
Yeah, that was the original reason for this issue. Other than the faint possibility of someone maybe needing this, this is really quite a confusing way of working. I can’t really point out another web app which does it like that.

**Comment by YaManicKill** [comment]
I am definitely for syncing settings across, but I would also want local overrides. I'm currently thinking of different ways of doing this.

**Comment by FryDay** [comment]
I am someone that DOES need separate settings in different browsers. At home, I use notifications, but at work I don't want messages popping up on my screen.

**Comment by jancborchardt** [comment]
@FryDay That’s already handled by browsers because it’s HTML5 Notifications. On your work machine you can then simply not allow them when your browser asks you for it. (Just like geolocation etc in other apps.)

**Comment by YaManicKill** [comment]
Yeah, but that's only 1 particular setting, and its a bit silly to base that on a browser feature. We can't rely on that.
